### PR TITLE
test(explorer): add end-to-end test for empty state

### LINKED ISSUE #12178: test(de): visualization should display empty state when no data is present

### PR DESCRIPTION
**Pull request by AlirieGray**
Closes #12178

This PR adds an end-to-end test that checks for the empty state in the Data Explorer visualization when the query entered returns no results.

  - [x] Rebased/mergeable
  - [x] Tests pass

### CODE REVIEW

File: ui/cypress/e2e/explorer.test.ts [hunk 1]
```diff
@@ -85,6 +85,28 @@ describe('DataExplorer', () => {
       cy.get('.input-field').type('covariance')
       cy.getByTestID('toolbar-function').should('have.length', 1)
     })
+
+    it.only('shows the empty state when the query returns no results', () => {
```
Comment (121watts) [inline ui/cypress/e2e/explorer.test.ts]: still have an only right there :)
